core/modules/qproc/QuerySession.h

```diff
@@ -79,8 +79,9 @@ class QuerySession {
 
     QuerySession(std::shared_ptr<css::CssAccess> const& css,
                  qproc::DatabaseModels::Ptr const& dbModels,
-                 std::string const& defaultDb)
-        : _css(css), _defaultDb(defaultDb), _databaseModels(dbModels) {}
+                 std::string const& defaultDb,
+                 int const interactiveChunkLimit)
```

Comment:
Extra `const` for parameters passed by value isn't strictly required. But it won't hurt either.